ansible/roles/debops.docker/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Usually there's an empty space here, but it's not a YAML convention or a requirement as far as I know.